Tests/ARTRealtimePresenceTest.m

```diff
@@ -486,18 +486,13 @@ -(void) testEnterAndGet {
             XCTAssertNil(errorInfo);
             [channel2.presence enter:enter2 callback:^(ARTErrorInfo *errorInfo) {
                 XCTAssertNil(errorInfo);
-
-                [channel2.presence get:^(ARTPaginatedResult *result, NSError *error) {
+                [channel2.presence get:^(NSArray<ARTPresenceMessage *> *members, ARTErrorInfo *error) {
                     XCTAssert(!error);
-                    NSArray *messages = [result items];
-                    XCTAssertEqual(2, messages.count);
-                    ARTPresenceMessage *m0 = messages[0];
-                    ARTPresenceMessage *m1 = messages[1];
-                    XCTAssertEqual(m0.action, ARTPresencePresent);
-                    XCTAssertEqual(m1.action, ARTPresencePresent);
-                    //TODO work out why the order seems to be random
-                    //XCTAssertEqualObjects(enter2, [m0 data]);
-                    //XCTAssertEqualObjects(enter, [m1 data]);
+                    XCTAssertEqual(2, members.count);
+                    XCTAssertEqual(members[0].action, ARTPresencePresent);
```

Comment:
Didn't you say here https://github.com/ably/ably-ios/pull/299#issuecomment-197005427 that the order isn't deterministic?